front/components/poke/PokeNavbar.tsx

```diff
@@ -31,6 +31,9 @@ export const PokeNavbar: React.FC = () => (
         <Link href="/poke/templates">
           <div className="text-stucture-300 text-sm italic">Templates</div>
         </Link>
+        <Link href="/poke/plugins">
+          <div className="text-stucture-300 text-sm italic">Plugins</div>
```

Comment:
`nit`: Could use `Label` here

Reply:
Our Sparkle component?

Reply:
yup

Reply:
Our `Label` component does not have any style when used in a hoverable state. Went with our `Button` in `ghost` variant.